File: 04-FEA-Truss/Matlab-Code/PrintStressStrain.m
function PrintStressStrain(stress,strain)

fout = fopen('Stress_Vector.dat','w');
fprintf(fout, '%12.8f \n', stress);
fclose(fout); 

fout = fopen('Strain_Vector.dat','w');
fprintf(fout, '%12.8f \n', strain);
fclose(fout); 

end

File: 04-FEA-Truss/C-Code/PrintStressStrain.cpp
#include <MainFile.h>
using namespace std;

void PrintStressStrain(arma::mat& strain, arma::mat& stress)
{

	ofstream fout;

  fout.open ("Stress_Vector.dat");
  fout << stress;
  fout.close();
  
  fout.open ("Strain_Vector.dat");
  fout << strain;
  fout.close();
  

}
